config/zenoh-bridge-ros2dds-conf.json5

```diff
@@ -11,8 +11,6 @@
             "/control/command/hazard_lights_cmd",
             "/api/external/get/vehicle/status",
```

Comment:
Could you add a comment above these topics to tell users they are used by FMS?

Reply:
Comment added